test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -817,6 +822,58 @@ def check_infiltration(prototypes, prototypes_base)
     end
   end
 
+  # Check if the hvac baseline system from 5 to 13 has the HW and CHW reset control
+  # Expected outcome
+  #@param prototypes_base[Hash] Baseline prototypes
+  def check_hw_chw_reset(prototypes_base)
+
+    # check if the numbers are correct
+    chw_low_temp = 15.5
+    chw_low_temp_reset = 12.2
+    chw_high_temp = 26.7
+    chw_high_temp_reset = 6.6
+    hw_low_temp = -6.7
+    hw_low_temp_reset = 82.2
+    hw_high_temp = 10.0
+    hw_high_temp_reset = 65.5
```

Comment:
Not a big deal but it'd be nice to use `OpenStudio.convert()` to go from the values in 90.1 which are in IP to values in SI. It'd make this and future code reviews a bit easier. No need to do it here since it's just a test.